src/mscorlib/model.xml

```diff
@@ -1793,31 +1793,122 @@
+      <Member MemberType="Field" Name="Cookies" />
+      <Member MemberType="Field" Name="Desktop" />
+      <Member MemberType="Field" Name="Favorites" />
+      <Member MemberType="Field" Name="History" />
+      <Member MemberType="Field" Name="InternetCache" />
+      <Member MemberType="Field" Name="Programs" />
+      <Member MemberType="Field" Name="MyComputer" />
+      <Member MemberType="Field" Name="MyMusic" />
+      <Member MemberType="Field" Name="MyPictures" />
+      <Member MemberType="Field" Name="MyVideos" />
+      <Member MemberType="Field" Name="Recent" />
+      <Member MemberType="Field" Name="SendTo" />
+      <Member MemberType="Field" Name="StartMenu" />
+      <Member MemberType="Field" Name="Startup" />
+      <Member MemberType="Field" Name="System" />
+      <Member MemberType="Field" Name="Templates" />
+      <Member MemberType="Field" Name="DesktopDirectory" />
+      <Member MemberType="Field" Name="Personal" />
+      <Member MemberType="Field" Name="MyDocuments" />
+      <Member MemberType="Field" Name="ProgramFiles" />
+      <Member MemberType="Field" Name="CommonProgramFiles" />
+      <Member MemberType="Field" Name="AdminTools" />
+      <Member MemberType="Field" Name="CDBurning" />
+      <Member MemberType="Field" Name="CommonAdminTools" />
+      <Member MemberType="Field" Name="CommonDocuments" />
+      <Member MemberType="Field" Name="CommonMusic" />
+      <Member MemberType="Field" Name="CommonOemLinks" />
+      <Member MemberType="Field" Name="CommonPictures" />
+      <Member MemberType="Field" Name="CommonStartMenu" />
+      <Member MemberType="Field" Name="CommonPrograms" />
+      <Member MemberType="Field" Name="CommonStartup" />
+      <Member MemberType="Field" Name="CommonDesktopDirectory" />
+      <Member MemberType="Field" Name="CommonTemplates" />
+      <Member MemberType="Field" Name="CommonVideos" />
+      <Member MemberType="Field" Name="Fonts" />
+      <Member MemberType="Field" Name="NetworkShortcuts" />
+      <Member MemberType="Field" Name="PrinterShortcuts" />
+      <Member MemberType="Field" Name="UserProfile" />
+      <Member MemberType="Field" Name="CommonProgramFilesX86" />
+      <Member MemberType="Field" Name="ProgramFilesX86" />
+      <Member MemberType="Field" Name="Resources" />
+      <Member MemberType="Field" Name="LocalizedResources" />
+      <Member MemberType="Field" Name="SystemX86" />
+      <Member MemberType="Field" Name="Windows" />
+    </Type>
     <Type Name="System.Environment">
+      <Member Name="get_CommandLine"  />
+      <Member Name="get_CurrentDirectory"  />
       <Member Name="get_CurrentManagedThreadId"  />
+      <Member Name="get_ExitCode"  />
+      <Member Name="get_HasShutdownStarted"  />
+      <Member Name="get_Is64BitProcess"  />
+      <Member Name="get_Is64BitOperatingSystem"  />
       <Member Name="get_MachineName"  />
+      <Member Name="get_NewLine"  />
       <Member Name="get_OSVersion" />
```

Comment:
@weshaggard: It breaks with error message `OSVersion's call to GetVersionEx failed.`, because an attempt is made to call _Windows_ API functions: https://github.com/dotnet/coreclr/blob/cdff8b0babe5d82737058ccdae8b14d8ae90160d/src/mscorlib/src/System/Environment.cs#L398